presentation/components/src/presentation-components/properties/UseNavigationPropertyTargetsLoader.ts

```diff
@@ -46,7 +46,7 @@ export function useNavigationPropertyTargetsLoader(props: UseNavigationPropertyT
       keys: new KeySet(),
       descriptor: {
         contentFlags: ContentFlags.ShowLabels | ContentFlags.NoFields,
-        filterExpression: filter ? `/DisplayLabel/ LIKE \"%${filter}%\"` : undefined,
+        fieldsFilterExpression: filter ? `/DisplayLabel/ LIKE \"%${filter}%\"` : undefined,
```

Comment:
I suppose we don't have a test for this? Should be using `~` instead of `LIKE`

Reply:
Changed to `~`